pkg/catalog/catalog.go

```diff
@@ -71,7 +73,11 @@ func (c *Catalog) GetWithVersion(sourceName, profileName, profileVersion string)
 	}
 
 	if profileVersion == "latest" {
-		return getLatestVersion(profiles.([]profilesv1.ProfileDescription), profileName)
+		versions := c.GetGreaterThan(sourceName, profileName, profileVersion)
```

Comment:
```suggestion
		versions := c.ProfileGreaterThanVersion(sourceName, profileName, profileVersion)
```

for consistency with the handler? its also more descriptive :smile:

Reply:
I'm not sure I follow this one? This is inside catalog, has nothing to do with the handler. We are consistent with the names in this file and not another. :D

Reply:
I meant more that `GetGreaterThan` inside `catalog` package is sort of vague.. greater than what? `ProfilesGreaterThanVersion` is much cleaner on what it returns

Reply:
Oh, ooooh I see. I left out version there. :D Why did I do that? :D